genkit-tools/cli/src/commands/eval-flow.ts

```diff
@@ -50,7 +52,10 @@ export const evalFlow = new Command('eval:flow')
   )
   .argument('<flowName>', 'Name of the flow to run')
   .argument('[data]', 'JSON data to use to start the flow')
-  .option('--input <filename>', 'JSON batch data to use to run the flow')
+  .option(
+    '--input <input>',
+    'Input dataset ID or JSON file to be used for evaluation'
```

Comment:
nit: Maybe a separate --datasetid option could be more intuitive to use.

Reply:
I think the versatility helps. We have to rethink the args for CLI commands anyway. We can incorporate this there. Thanks!